Cargo.toml

```diff
@@ -7,6 +7,7 @@ description = "Audio playback library"
 keywords = ["audio", "playback", "gamedev"]
 repository = "https://github.com/RustAudio/rodio"
 documentation = "http://docs.rs/rodio"
+autoexamples = true
```

Comment:
This is redundant as true is already the default

Reply:
In fact I am not even sure if setting it to `true` have any effect...
But at least, it remove a warning.

Since I am setting an example like:
```
[[example]]
name = "http_flac"
path = "examples/http_flac.rs"
required-features = ["http"]
```

I got a warning:
```
warning: An explicit [[example]] section is specified in Cargo.toml which currently
disables Cargo from automatically inferring other example targets.
This inference behavior will change in the Rust 2018 edition and the following
files will be included as a example target:

* /home/tirz/Documents/Rust/rodio/examples/basic.rs
* /home/tirz/Documents/Rust/rodio/examples/music_mp3.rs
* /home/tirz/Documents/Rust/rodio/examples/music_flac.rs
* /home/tirz/Documents/Rust/rodio/examples/reverb.rs
* /home/tirz/Documents/Rust/rodio/examples/music_wav.rs
* /home/tirz/Documents/Rust/rodio/examples/music_ogg.rs
* /home/tirz/Documents/Rust/rodio/examples/spatial.rs

This is likely to break cargo build or cargo test as these files may not be
ready to be compiled as a example target today. You can future-proof yourself
and disable this warning by adding `autoexamples = false` to your [package]
section. You may also move the files to a location where Cargo would not
automatically infer them to be a target, such as in subfolders.

For more information on this warning you can consult
https://github.com/rust-lang/cargo/issues/5330
```